README.rst

```diff
@@ -139,6 +141,10 @@ ev3python.com
     If you are just getting started with programming, we highly recommend
     that you check it out at `ev3python.com`!
 
+Frequently-Asked Questions
```

Comment:
It _should_ be part of the list thing that the other items are in... I have no idea what it's called :laughing: